src/_includes/banner.html

```diff
@@ -2,7 +2,7 @@
 <div class="banner">
   <p class="banner__text">
     {% comment %} TEXT GOES HERE {% endcomment %}
-    Dart 2.5.0: Super-charged development with ML Complete and dart:ffi! For more information, see the
-    <a class="no-automatic-external" href="https://medium.com/dartlang/announcing-dart-2-5-super-charged-development-328822024970">blog post</a>.
+    Announcing verified publishers for pub.dev! For more information, see the
```

Comment:
How about just making the first sentence a link to the blog post? Or maybe add a link to the verified publishers page, with the blog post the second thing. Possibilities:

<a href="https://medium.com/dartlang/verified-publishers-98f05466558a">Announcing verified publishers for pub.dev!</a>

OR

Announcing <a href="/tools/pub/verified-publishers">verified publishers</a> for pub.dev! For details, see the <a href="https://medium.com/dartlang/verified-publishers-98f05466558a">announcement.</a>

Reply:
Great idea, please take another look